lgsm/functions/command_install_init.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# LinuxGSM command_install_init.sh function
+# Author: legendofmiracles
+# Website: https://linuxgsm.com
+# Description: Installs the service files for systemd in the correct place.
+commandname="INITSYSTEM"
+commandaction="systemd service file"
+functionselfname="$(basename "$(readlink -f "${BASH_SOURCE[0]}")")"
+fn_firstcommand_set
+
+fn_install_systemd() {
+	file_c=$'[Unit]
+Description=LinuxGSM gamename Server
+After=network-online.target
+Wants=network-online.target
+
+[Service]
+Type=forking
+User=whoami
+ExecStart=path start
+ExecStop=path stop
+Restart=no
+RemainAfterExit=yes
+
+[Install]
+WantedBy=multi-user.target'
+	file_c=$(printf "$file_c" | sed s/'gamename'/$gamename/g | sed "s|path|$path|g" | sed s/'whoami'/$(whoami)/g)
+	fn_print_ok_nl "Generated the file contents"
+	fn_print_information_nl "Enter the password of root:"
+	su -c "echo \"$file_c\" > /etc/systemd/system/${selfname}-lgsm.service" || return 1 && return 0
```

Comment:
I am having issues getting this to work. 

The only other time LinuxGSM tries to elevate to root is using `sudo` privileges for a user. However this is not always possible. I would suggest if the user is unable to elevate that a file is generated and the command is printed to screen so the user can do it manually from root.

Reply:
I am looking forward to getting this feature added. It is going to be a great addition to LinuxGSM :)

Reply:
So it should use sudo instead of su, and check if sudo failed in order to print the contents and instruct the user how/where to place those?

Reply:
It now uses sudo and prints the contents on error - with instructions where to put it.

Reply:
edit: i just saw that you wanted a command printed - will do